src/bikeracks/schema.sql

```diff
@@ -74,7 +74,8 @@ CREATE TABLE suggested_removals (
     rack_id INTEGER NOT NULL,
     user_id TEXT,
     time_stamp INTEGER,
-    reason_id INTEGER
+    reason_id INTEGER,
+    CHECK (reason_id in (1, 2))
```

Comment:
This is what the foreign key constraint is supposed to ensure.

All and only the valid reasons should be in that table.